integration/monai/flare_monai_integration.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
     "<img src=\"images/MONAI-FL.svg\" alt=\"MONAI FL Integration\" width=400/>\n",
     "\n",
-    "You can find the code for the [MONAI Integration code](https://github.com/NVIDIA/NVFlare/blob/dev/integration/monai/README.md) in the `integration` directory of the NVFlare GitHub.\n",
+    "You can find the code for the [MONAI Integration code](https://github.com/NVIDIA/NVFlare/blob/main/integration/monai/README.md) in the `integration` directory of the NVFlare GitHub.\n",
     "\n",
-    "There is also an example that walks through preparing the environment and input datasets, and running the [MONAI Spleen CT Segmentation example](https://github.com/NVIDIA/NVFlare/tree/dev/integration/monai/examples/spleen_ct_segmentation_real-world) using a locally provisioned secure deployment.  We'll walk through that here."
+    "There is also an example that walks through preparing the environment and input datasets, and running the [MONAI Spleen CT Segmentation example](https://github.com/NVIDIA/NVFlare/tree/main/integration/monai/examples/spleen_ct_segmentation_real-world) using a locally provisioned secure deployment.  We'll walk through that here."
```

Comment:
This job no longer exist.
Either local or sim

Reply:
changed to _local